src/routes/PaymentResponsePageV2.tsx

```diff
@@ -66,25 +87,14 @@ export default function PaymentResponsePageV2() {
     getCartReturnUrl(outcome)
   );
 
-  const transactionData = getSessionItem(SessionItems.transaction) as
-    | NewTransactionResponse
-    | undefined;
-
-  const pspSelected = getSessionItem(SessionItems.pspSelected) as
-    | Bundle
-    | undefined;
-
   const email = getSessionItem(SessionItems.useremail) as string | undefined;
 
-  const totalAmount =
-    Number(
-      // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
-      transactionData?.payments.reduce((sum, { amount }) => sum + amount, 0)
-    ) + Number(pspSelected?.taxPayerFee);
+  const grandTotalAmount =
```

Comment:
in order to consider 0 or 0,00 as valid value
```suggestion
const grandTotalAmount =
  totalAmount != null && fees != null
    ? Number(totalAmount) + Number(fees)
    : null;
```

Reply:
Thanks, that was a mistake, fixed with [c476e36](https://github.com/pagopa/pagopa-checkout-fe/pull/508/commits/c476e36f1add067e6a047ebb94e8af922d4de10f)